understanding/20/bypass-blocks.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Bypass Blocks</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Users can more easily navigate by keyboard</dd>
-		<dt>Author task</dt><dd>Provide a means of skipping repeating content</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Users can more easily navigate by keyboard.</dd>
+         <dt>What to do</dt><dd>Provide a means of skipping repeating content.</dd>
```

Comment:
I think it is not about repeating content, but content which is grouped in blocks.